src/tasks/__init__.py

```diff
@@ -14,10 +14,15 @@
 from .tasks import Task
 
 ##
+# All cola-like tasks for which we need write_preds() and append to ALL_GLUE_TASKS below
+ALL_COLA_NPI_TASKS = ['cola_npi_x']
+
+
 # Task lists for handling as a group; these names correspond to the keys in
 # the task registry.
 ALL_GLUE_TASKS = ['sst', 'cola', 'mrpc', 'qqp', 'sts-b',
-                  'mnli', 'qnli', 'rte', 'wnli', 'mnli-diagnostic']
+                  'mnli', 'qnli', 'rte', 'wnli', 'mnli-diagnostic'] + ALL_COLA_NPI_TASKS
```

Comment:
Remove this—it'll impact any experiment anyone outside does with GLUE.

Reply:
Done. Relocated that to `evaluate.py`